### PR TITLE
Feature/17 UI for create users  3

### PR DESCRIPTION
**Pull request by philipkcl**
Fix admin page create user fail .

Related issues:
* https://github.com/culturesofknowledge/emlo-project/issues/20
* https://github.com/culturesofknowledge/emlo-project/issues/21
* https://github.com/culturesofknowledge/emlo-project/issues/17

### CODE REVIEW

File: login/forms.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+from django.contrib.auth.forms import UserCreationForm, UserChangeForm
+
+from login.models import CofkUser
+
+
+class CustomUserCreationForm(UserCreationForm):
```
Comment (J4bbi) [inline login/forms.py:6]: `CofkUserCreationForm`

**Review by J4bbi — CHANGES_REQUESTED** [review]
At this stage I don´t think we need a custom `UserManager`, since we are still using the same main user properties as the default user (email and passw).

However, the admin user change form don´t hash the password. @philipkcl , could you either follow the suggestions I posted as a comment [here](https://github.com/culturesofknowledge/site-edit-2/pull/2/files/a7275e840834d3de3a3012f49c6953b9ae0a1224#diff-f906c2dd0da247d64b767588df0542d6d86bbb11c1d30bfc19753123aa82cf1b), or perhaps use [AdminPasswordChangeForm](https://docs.djangoproject.com/en/4.0/topics/auth/default/#django.contrib.auth.forms.AdminPasswordChangeForm) ?

File: login/admin.py [hunk 2]
```diff
@@ -1,7 +1,39 @@
...
+from django.contrib.auth.admin import UserAdmin
 
 # Register your models here.
-
+from .forms import CustomUserCreationForm, CustomUserChangeForm
 from .models import CofkUser
 
-admin.site.register(CofkUser)
+
+class CustomUserAdmin(UserAdmin):
```
[the remark this answers is not in the document]
Comment (philipkcl) [reply login/admin.py:9]: but every class with prefix Cofk could be redundant, some time is also make confuse with models class, I would like to keep the `CustomUserAdmin`

**Comment by philipkcl** [comment]
> At this stage I don´t think we need a custom `UserManager`, since we are still using the same main user properties as the default user (email and passw).
> 
> However, the admin user change form don´t hash the password. @philipkcl , could you either follow the suggestions I posted as a comment [here](https://github.com/culturesofknowledge/site-edit-2/pull/2/files/a7275e840834d3de3a3012f49c6953b9ae0a1224#diff-f906c2dd0da247d64b767588df0542d6d86bbb11c1d30bfc19753123aa82cf1b), or perhaps use [AdminPasswordChangeForm](https://docs.djangoproject.com/en/4.0/topics/auth/default/#django.contrib.auth.forms.AdminPasswordChangeForm) ?

tested `admin user change form` is hashed the password now. Could you show me the step that making non hashed password.

**Pull request merged**